atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/ImportRenderer.java

```diff
@@ -25,7 +25,7 @@
 import com.google.common.collect.Iterables;
 
 public class ImportRenderer extends Renderer {
-    private static final List<String> IMPORT_PREFIXES = ImmutableList.of("java.", "javax.", "org.", "com.");
+    private static final List<String> IMPORT_PREFIXES = ImmutableList.of("java.", "javax.", "org.", "com.", "net.");
```

Comment:
Tracked it down: #1413.
Will merge develop into this branch, then fix a minor indentation issue, then generate schemas, then merge.